frontend/src/metabase/components/YearPicker/YearPicker.unit.spec.tsx

```diff
@@ -6,6 +6,6 @@ import YearPicker from "./YearPicker";
 describe("YearPicker", () => {
   it("should render correctly", () => {
     render(<YearPicker value={2022} onChange={jest.fn()}></YearPicker>);
-    expect(screen.getByRole("button")).toHaveTextContent("2022");
+    expect(screen.getByTestId("select-button")).toHaveTextContent("2022");
```

Comment:
@ariya I modified this bit otherwise, the test would fail https://github.com/metabase/metabase/runs/6406039725?check_suite_focus=true.

My whole PR is changing the `Select` component which `YearPicker` renders.